### PR TITLE
Josue/add sticky menu

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
This PR completes [Implement sticky-menu section](https://app.shortcut.com/clarkcan/story/12463/implement-sticky-menu-section)

*Note*: In mobile view, the menu is removed.

Screenshots:
Screen Shot 2022-08-10 at 11 26 24 AM

Video of **Updated** scrolling:

https://user-images.githubusercontent.com/72173743/184241899-9c6ff3ad-3184-440f-a80f-3041acd69ad2.mov

### CODE REVIEW

File: src/app/cube/new-home/learning-object-info/sticky-menu/sticky-menu.component.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        border-radius: 5px;
+        font-size: large;
+        width: 230px;
+        margin: 5px;
+        .icon {
+            font-size: 20px;
+            margin-right: 10px;
+        }
+        &:hover {
+            background-color: #E9F0FE;
```
Comment (EVogel1999) [inline src/app/cube/new-home/learning-object-info/sticky-menu/sticky-menu.component.scss:25]: Is there a way we can make these variables either in this file or in the vars.scss file?

**Review by EVogel1999 — COMMENTED** [review]
Just one comment about turning some css hex values into variables instead.

DLnwGUSWsAEKEwb

**Review by AStoerkel1 — APPROVED** [review]
Looks good

**Comment by katieblomquist** [comment]
This is looking great! The transitions do feel a little jarring. Maybe consider adding an animation to transition to and from activated elements?

**Review by jouypo23 — APPROVED** [review]
Looks great! Beyond Katelyn's comment, this looks sleek.

**Review by pmanal1 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]

**Review by katieblomquist — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]

**Review by pmanal1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by AStoerkel1 — APPROVED** [review]

**Pull request merged**